python/python/lancedb/query.py

```diff
@@ -1150,7 +1129,15 @@ def refine_factor(self, refine_factor: int) -> LanceHybridQueryBuilder:
         LanceHybridQueryBuilder
             The LanceHybridQueryBuilder object.
         """
-        self._vector_query.refine_factor(refine_factor)
+        self._refine_factor = refine_factor
+        return self
+
+    def vector(self, vector: Union[np.ndarray, list]) -> LanceHybridQueryBuilder:
```

Comment:
NIT: it can be a string right?

Reply:
No I think this one can't be string. This is meant for cases where user wants to explicitly pass the vector without relying on our embedding API

Reply:
So if the users want to use our embedding API, it's
```
table.search(vec_string).text(fts_query)
```
right?

Reply:
With embedding API the string will automatically be embedded and it will also be used for fts. 
'Search(text_query, query_type="hybrid")'
This is the current usage. 
With the new API, we'll allow passing vector and text queries separately.